airbyte-integrations/connectors/destination-e2e-test/acceptance-test-config.yml

```diff
@@ -0,0 +1,4 @@
+connector_image: airbyte/destination-e2e-test:dev
+acceptance_tests:
+  spec:
+    tests: [{}]
```

Comment:
Empty test object is necessary otherwise fixtures are missing and the test can't run. Could be filled in via default.